packages/flutter_driver/lib/src/driver/scene_display_lag_summarizer.dart

```diff
@@ -35,41 +35,41 @@ const String _kVsyncTransitionsMissed = 'vsync_transitions_missed';
 class SceneDisplayLagSummarizer {
   /// Creates a SceneDisplayLagSummarizer given the timeline events.
   SceneDisplayLagSummarizer(this.sceneDisplayLagEvents) {
-    for (final TimelineEvent event in sceneDisplayLagEvents) {
+    for (final TimelineEvent event in sceneDisplayLagEvents!) {
```

Comment:
Why's the final field sceneDisplayLagEvents nullable in the first place then? Wouldn't this always crash when you instantiate SceneDisplayLagSummarizer(null)?

Reply:
Yes, updating this. I think we just were missing an assert there.. Making it non-null.